File: backend/master/src/main/java/com/ociet/loadgenerator/master/MasterApplication.java
package com.ociet.loadgenerator.master;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.server.EnableEurekaServer;

@SpringBootApplication
@EnableEurekaServer
public class MasterApplication {

    public static void main(String[] args) {
        SpringApplication.run(MasterApplication.class, args);
    }

}

File: backend/master/src/main/kotlin/com/chylb/loadgenerator/master/MasterApplication.kt
package com.chylb.loadgenerator.master

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.netflix.eureka.server.EnableEurekaServer

@SpringBootApplication
@EnableEurekaServer
open class MasterApplication

fun main(args: Array<String>) {
    SpringApplication.run(MasterApplication::class.java, *args)
}